h2integrate/converters/grid/grid_sell.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import openmdao.api as om
+from attrs import field, define
+
+from h2integrate.core.utilities import BaseConfig, CostModelBaseConfig, merge_shared_inputs
+from h2integrate.core.model_baseclasses import CostModelBaseClass
+
+
+@define
+class GridSellPerformanceModelConfig(BaseConfig):
+    """Configuration for grid electricity selling performance model."""
+
+    interconnect_limit: float = field(default=1000000.0)  # kW
+
+
+class GridSellPerformanceModel(om.ExplicitComponent):
```

Comment:
I think it could be more clear to have buy and sell capabilities in a single grid component, both to mirror the actual hardware in the system and because I think that would be more intuitive to the user.